images/tags.md

```diff
@@ -38,3 +40,53 @@ To use an existing tag as the default tag, click the **vertical ellipses** for a
 tag and select **Make default**.
 
 ![Set Existing Tag as Default](../assets/existing-tag-as-default.png)
+
+## Image tag lifecycle
+
+Coder uses image tags to determine which image variant on which the user wants
+to base their environment.
+
+Image tags are expressed using the following notation:
+
+```text
+<image>:<image-tag>
+```
+
+Examples include:
+
+```text
+ubuntu:20.04
+ubuntu:rolling
+codercom/enterprise-node:ubuntu
+mycorp/myproject:v1
+```
+
+The name formatting of the tag will change the way Coder behaves regarding image
+updates:
+
+- If you build your environment using a `ubuntu:rolling` or `ubuntu:latest` tag,
+  Coder prompts you to rebuild for patches, security updates, and major version
+  releases. If you're supporting a SaaS product or working on mobile apps, you
+  may opt for this to ensure that your tools stay up-to-date.
+
+- If you build your environment using a specific version tag (e.g.,
+  `ubuntu:20.04`), Coder will alert you regarding patches and security updates
+  so that you rebuild your environment (you won't get these fixes otherwise).
+  Coder does not, however, alert you regarding minor releases (e.g., movement
+  from `20.04` to `20.10`). This is a good option for those offering long-term
+  support of software with lengthier version cycles or those supporting multiple
+  versions where you expect to revert back to a prior release to investigate and
+  fix issues.
+
+- If you build your environment using a tag like
+  `codercom/enterprise-node:ubuntu`, the image name indicates the company and
```

Comment:
The `codercom` organization contains all of our example images, `enterprise` refers to Coder Enterprise (and should probably be rebranded), and `node` refers to the fact that the image contains the build tooling necessary for building Node.js applications. The names are all arbitrary though.